tests/pytorch/mock_sparse/test_elementwise_op_sp.py

```diff
@@ -36,12 +43,35 @@ def _test():
             all_close_sparse(op(A.adj, B.adj), op(A, B).adj)
         else:
             # sparse div is not supported in PyTorch
-            assert np.allclose(op(A, A1).val, op(A.val, A1.val), rtol=1e-4, atol=1e-4)
+            assert np.allclose(
+                op(A, A1).val, op(A.val, A1.val), rtol=1e-4, atol=1e-4
+            )
+
+    _test()
+
+
+@parametrize_idtype
+@parametrize_dtype
+# TODO (Israt): Add div operator or not. Div does not support matrices with
```

Comment:
You can have a separate test case for div.

Reply:
Done.